src/routes/index.tsx

```diff
@@ -3,13 +3,15 @@ import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
 import Login from 'screens/Login';
 import SignUp from 'screens/SignUp';
+import Home from 'screens/Home';
 import { ROUTES } from 'constants/index';
 
 function Routes() {
   return (
     <Router>
       <Switch>
         <Route path={ROUTES.signUp} component={SignUp} />
```

Comment:
You can improve your const Routes adding path, component and other properties. And here you can loop it for create dynamic routes
```
Routes.map(({component, path , exact}) => <Route path={path} component={component} />)

```